### DISCUSSION TITLE
Environment variable for CA certificate location in Kafka client

### DISCUSSION THREAD

**Discussion by prashantochatterjee** [confluentinc/confluent-kafka-python#1359] [Q&A]
Is there any environment variable that I can use to specify the location of the CA certificate instead of the property `ssl.ca.location`?

**Comment by OneCricketeer** [comment]
What's stopping you from providing `'ssl.ca.location': os.environ['YOUR_KEY']`?

**Reply by pchatterjee-cfx** [reply]
Agree @OneCricketeer. In hindsight, I could have used it. I was in fact looking for CA PEM certificate string format that was added in 2.0.0. librdkafka release.
https://github.com/confluentinc/librdkafka/releases/tag/v2.0.0
